plugins/builds.js

```diff
@@ -87,11 +93,23 @@ exports.plugin = {
                     }
                 }
 
+                const fileName = artifact.split('/').pop();
```

Comment:
@tkyi @adong can add the encodeURI here. And change this line to use fileName: https://github.com/screwdriver-cd/store/blob/00fa98be5b377907d32baa2870ddbe25aa9b7a69/plugins/builds.js#L104

Reply:
Sounds good.